include/hobbes/net.H

```diff
@@ -1166,17 +1166,17 @@ template <typename ... Args>
     uint32_t exprid;
   };
 
-#define _HNET_CLIENT_MAKE_EXPRID(n, _, __) , exprID_##n
-#define _HNET_CLIENT_MAKE_RPCDEF(n, t, e) result.push_back(::hobbes::net::RPCDef((uint32_t)exprID_##n, e, ::hobbes::net::RPCTyDef<t>::inputType(), ::hobbes::net::RPCTyDef<t>::outputType()));
-#define _HNET_CLIENT_INIT_RPCFUNC(n, t, _) , n(&this->s, (uint32_t)exprID_##n)
-#define _HNET_CLIENT_MAKE_RPCFUNC(n, t, _) ::hobbes::net::RPCFunc<t> n;
+#define PRIV_HNET_CLIENT_MAKE_EXPRID(n, _, __) , exprID_##n
```

Comment:
* The use of [this double underscore for a macro parameter](https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier#answer-228797 "What are the rules about using an underscore in a C++ identifier?") might be also questionable.
* How much do macros make sense where a parameter is not used so far?